packages/host/src/components/Navigation/RequireAuthentication.tsx

```diff
@@ -1,17 +1,18 @@
 import {
   Navigate,
-  useLocalStorage,
+  useAuthContext,
   useLocation,
 } from '@openmsupply-client/common';
 import { AppRoute } from '@openmsupply-client/config';
 import React from 'react';
 
 export const RequireAuthentication = ({ children }) => {
-  const [authToken] = useLocalStorage('/authentication/token');
+  const { token } = useAuthContext();
   const location = useLocation();
-  const isProduction = process.env['NODE_ENV'] === 'production';
+  // const isProduction = process.env['NODE_ENV'] === 'production';
```

Comment:
I thought it might be useful to be logged out, even when developing, now that the API requires valid credentials for most things, and there isn't a mock server for dev work

Reply:
Yes, it's actually a little confusing to be logged in and not be logged in!